file/provider/create_volume.go

```diff
@@ -40,27 +38,37 @@ func (vpcs *VPCSession) CreateVolume(volumeRequest provider.Volume) (volumeRespo
 	defer vpcs.Logger.Debug("Exit from CreateVolume method...")
 	defer metrics.UpdateDurationFromStart(vpcs.Logger, "CreateVolume", time.Now())
 
+	var iops int64
+	var bandwidth int32
 	vpcs.Logger.Info("Basic validation for CreateVolume request... ", zap.Reflect("RequestedVolumeDetails", volumeRequest))
-	resourceGroup, iops, err := validateVolumeRequest(volumeRequest)
+	resourceGroup, iops, bandwidth, err := validateVolumeRequest(volumeRequest)
 	if err != nil {
 		return nil, err
 	}
+
 	vpcs.Logger.Info("Successfully validated inputs for CreateVolume request... ")
 
+	// Set zone if provided
+	var zone *models.Zone
+	if volumeRequest.Az != "" {
```

Comment:
this may be a issue if anyone using this library and passing zone in case of RFS profile, we will make a call to RIaaS which will fail, as we are already checking in CSI driver so we are good with this change